android/src/main/java/io/sentry/react/RNSentryReplayBreadcrumbConverter.java

```diff
@@ -0,0 +1,37 @@
+package io.sentry.react;
+
+import io.sentry.Breadcrumb;
+import io.sentry.android.replay.DefaultReplayBreadcrumbConverter;
+import io.sentry.rrweb.RRWebEvent;
+import io.sentry.rrweb.RRWebBreadcrumbEvent;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class RNSentryReplayBreadcrumbConverter extends DefaultReplayBreadcrumbConverter {
+  public RNSentryReplayBreadcrumbConverter() {
+  }
+
+  @Override
+  public @Nullable RRWebEvent convert(final @NotNull Breadcrumb breadcrumb) {
+    var rrwebBreadcrumb = new RRWebBreadcrumbEvent();
```

Comment:
```suggestion
    RRWebBreadcrumbEvent rrwebBreadcrumb = new RRWebBreadcrumbEvent();
```
I think `var` will fail for AGP < 7.4.0 (where Java 11 features were introduced through desugaring), not sure what's the min supported AGP version for RN, but better be safe I guess

Reply:
Currently we support RN 0.65 -> that means Gradle 6.9. 

https://react-native-community.github.io/upgrade-helper/?from=0.65.3&to=0.74.1